Makefile

```diff
@@ -27,6 +28,16 @@ build_name = $(COMMAND)-$(VERSION)-$(os)-amd64
 build_binary = $(build_name)$(BINARY_EXT_$(os))
 package_binary = $(COMMAND)$(BINARY_EXT_$(os))
 
+.PHONY: deps
```

Comment:
I'm okay with adding the `dep` tool and an `ensure` target, but instead of a `deps` make target to install it, lets be consistent with how we install golangci-lint. That is, make a variable for DEP_BINARY and a make target for the binary itself. Then the ensure target can depend on the DEP_BINARY to install it without additional commands and only when needed.